lib/resource-mapper.js

```diff
@@ -8,12 +8,21 @@ const readdir = promisify(fs.readdir)
 // following the principles of the “sweet spot” discussed in
 // https://www.w3.org/DesignIssues/HTTPFilenameMapping.html
 class ResourceMapper {
-  constructor ({ rootUrl, rootPath, includeHost, defaultContentType }) {
+  constructor ({
+    rootUrl,
+    rootPath,
+    includeHost,
+    defaultContentType,
```

Comment:
This one can be assigned to `'application/octet-stream'` by default; it's the most generic content type.